CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## Contributing
+
+If you want to contribute to a project and make it better, your help is very welcome. Contributing is also a great way to learn more about social coding on Github, new technologies and and their ecosystems and how to make constructive, helpful bug reports, feature requests and the noblest of all contributions: a good, clean pull request.
+
+### How to make a clean pull request
+
+Look for a project's contribution instructions. If there are any, follow them.
+
+- Create a personal fork of the project on Github.
+- Clone the fork on your local machine. Your remote repo on Github is called `origin`.
```

Comment:
Could you add the commandline instructions to do this?
```
git clone https://github.com/your-username/zookeeper.git
```

Reply:
I have added this instruction to the updated CONTRIBUTING.md file.